docs/deployment/README.md

```diff
@@ -352,25 +242,28 @@ List of EarthData users you wish to have access to your dashboard application.
     - username: <user2>
 ```
 
-##### Configure EarthData application
+### Configure EarthData application
 
 The Cumulus stack is expected to authenticate with [Earthdata Login](https://urs.earthdata.nasa.gov/documentation). You must create and register a new application. Use the [User Acceptance Tools (UAT) site](https://uat.urs.earthdata.nasa.gov) unless you changed `urs_url` above. Follow the directions on [how to register an application.](https://wiki.earthdata.nasa.gov/display/EL/How+To+Register+An+Application).  Use any url for the `Redirect URL`, it will be deleted in a later step. Also note the password in step 3 and client ID in step 4 use these to replace `clientid` and `clientpassword` in the `.env` file in the next step.
 
-##### Set up an environment file:
+### Set up an environment file
 
-_If you're adding a new deployment to an existing configuration repository or re-deploying an existing Cumulus configuration you should skip to [Deploy the Cumulus Stack](#deploy-the-cumulus-stack), as these values should already be configured._
+_If you're adding a new deployment to an existing configuration repository or re-deploying an existing Cumulus configuration you should skip to [Deploy the Cumulus Stack](#deploy), as these values should already be configured._
 
-Copy `app/.env.sample to app/.env` and add CMR/earthdata client [credentials](#Credentials):
+Copy `app/.env.sample` to `app/.env` and add CMR/earthdata client [credentials](#credentials):
 
     CMR_PASSWORD=cmrpassword
     EARTHDATA_CLIENT_ID=clientid
     EARTHDATA_CLIENT_PASSWORD=clientpassword
+    TOKEN_SECRET=tokensecret
+
+The `TOKEN_SECRET` is a string value used for token refreshes when using the Cumulus dashboard. This can be any string, preferably a 256 character random string for security purposes.
```

Comment:
Couple changes here:

* I would say that it is a string used for signing and verifying JWTs issued by the API
* It only needs to be a 32 character (256 bit) random string. I might phrase it as "this _should be_ a 32 character string", since it is strongly recommended for security reasons